compiled_starters/c/README.md

```diff
@@ -0,0 +1,35 @@
+![progress-banner](https://codecrafters.io/landing/images/default_progress_banners/shell.png)
+
+This is a starting point for C solutions to the
+["Build Your Own Shell" Challenge](https://app.codecrafters.io/courses/shell/overview).
+
+_In this challenge, you'll build your own POSIX compliant shell that's capable
```

Comment:
I'm not sure where the `_` surrounding this paragraph came from - isn't present in the Redis challenge for example: 

<img width="695" alt="Screenshot 2024-06-05 at 11 47 10" src="https://github.com/codecrafters-io/build-your-own-shell/assets/3893573/13eb279b-7867-4512-80e6-87a8d69d0815">

@glokta1 can we check other challenges, make sure we're consistent across all? I don't think the italics help, best removed

Reply:
@rohitpaulk Yup, without italics everywhere else. Got confused by the formatting in the template README. Fixed.